estate/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from email.policy import default
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
```

Comment:
The name of a model should always be separated with dots, not underscore
```suggestion
    _name = "estate.property"
```